src/Generator/Generators/CSharp/CSharpCommentPrinter.cs

```diff
@@ -81,6 +81,17 @@ private static void GetCommentSections(this Comment comment, List<Section> secti
                 case DocumentationCommentKind.InlineContentComment:
                     break;
                 case DocumentationCommentKind.InlineCommandComment:
+                    var lastSection = sections.Last();
+                    var inlineCommand = (InlineCommandComment) comment;
+
+                    if (inlineCommand.CommandKind == CommentCommandKind.B)
+                        if (lastSection.Lines.Count > 0)
+                            lastSection.Lines[lastSection.Lines.Count - 1] +=$" <c>{inlineCommand.Arguments[0].Text}</c>";
```

Comment:
Extract the string to a local variable and reuse it in both conditional branches.